Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -72,7 +72,7 @@ function OnUpdate_AirlockDoor( furniture, deltaTime )
                 adjacentRooms[count] = tile.Room
             end
         end
-        if(ModUtils.Round(adjacentRooms[1].GetTotalGasPressure(),3) == ModUtils.Round(adjacentRooms[2].GetTotalGasPressure(),3)) then
+        if(math.abs(ModUtils.Round(adjacentRooms[1].GetTotalGasPressure(),3) - ModUtils.Round(adjacentRooms[2].GetTotalGasPressure(),3)) < 0.005 ) then
```

Comment:
The epsilon for for float comparison is drastically smaller than this, and I'm not entirely sure of a proper variable name for this... maybe tolerance?